dts/arm/st/f1/stm32f105Xc.dtsi

```diff
@@ -30,6 +30,23 @@
 			status = "disabled";
 			label = "OTGFS";
 		};
+
```

Comment:
Update commit messge

Reply:
Please could you expand on what you mean by this?